cabal.project

```diff
@@ -1,13 +1,13 @@
 packages: ./typed-protocols
           ./typed-protocols-cbor
           ./ouroboros-network-testing
-	  ./network-mux
-	  ./ouroboros-network
-	  ./ouroboros-consensus
-	  ./io-sim
-	  ./io-sim-classes
-	  ./ntp-client
-	  ./Win32-network
+          ./network-mux
+          ./ouroboros-network
+          ./ouroboros-consensus
+          ./io-sim
+          ./io-sim-classes
+          ./ntp-client
+          ./Win32-network
```

Comment:
I would squash this change into the right commit, so that we're not needlessly tainting the history of this file :slightly_smiling_face: But I guess you were planning on doing that before merging it.

Reply:
Yes, I'll do that.